cost/azure/reserved_instances/utilization/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v3.0
+
+- Updated policy to move away from Azure Legacy EA API endpoints and use the Azure Modern API endpoints due to deprecation in 2024
+- Updated credential required for policy - it now requires `azure_auth` credential due to EA Key deprecation in 2024
```

Comment:
Names of credentials, parameters, etc. in the CHANGELOG should match what the user would see, not how things are identified in code. For example, instead of `azure_auth`, we can say it now requires an Azure Resource Manager credential, and we should replace `param_enrollment_id` with the name of the parameter the user would see in the UI.

Reply:
Resolved (I think)